versioned_docs/version-20.04/api/clapi.md

```diff
@@ -2362,6 +2362,8 @@ Parameters that you can change are the following:
 | autologin\_key        | Used for auto login                                                                  |
 | template              | Name of the template to apply to the contact                                         |
 | timezone              | Timezone                                                                             |
+| reach_api             | Reach API Configuration                                                              |
```

Comment:
```suggestion
| reach_api             | 1 if the user has access to API configuration, 0 otherwise                                                         |
```